xlsxwriter/worksheet.py

```diff
@@ -55,12 +56,12 @@ def cell_wrapper(self, *args, **kwargs):
         try:
             # First arg is an int, default to row/col notation.
             if len(args):
-                int(args[0])
+                first_arg = args[0]
```

Comment:
This change need more explanation.

Reply:
you call args[0] twice in some instances, due to the pervasive nature of this decorator, every nanosecond needs to be trimmed.  also, tuples are faster than lists, and you don't need to cast to a list if you use the plus operator.